src/Components/Header/Header.scss

```diff
@@ -1,5 +1,5 @@
 header {
-  background: linear-gradient(to top left, #FF0000 4%, #CC0000 100%);
+  background: #FF0000;
```

Comment:
Whats the logic for making the colors more flat?

Reply:
So the header style matches the pokedex style